concepticondata/conceptlists.tsv

```diff
@@ -18,8 +18,8 @@ Kingsada-1999-303	Kingsada, Thongpheth and Shintani, Tadahiko	1999		303	basic	En
-Swadesh-1964-215	Swadesh, Morris	1964		215	basic	Russian,English	Global	http://www.classes.ru/grammar/148.new-in-linguistics-1/source/worddocuments/_24.htm	Swadesh1964		This is the first Russian translation of the master list of [Swadesh's paper from 1955](:bib:Swadesh1955), which turned out to be very influential in Soviet linguistics and has retained influential up to today. Note that while the translation of the concept labels is literal and also provides the original ones which Swadesh provided in his c[master list of 215 concepts](:ref:Swadesh-1955-215), the Russian translation makes certain distinctions which cannot be determined from the original text. In one case, in the case of “fat”, this distinction is essentially wrong, as Swadesh did explicitly annotate this concept as a body part or substance. The concepts whose English labels are preceded by an asterisk are those 92 concepts which were later included in the famous [100-item list](:ref:Swadesh-1955-100).	78-87	
+Swadesh-1964-100	Swadesh, Morris	1964		100	basic	English,Russian	Global	http://www.classes.ru/grammar/148.new-in-linguistics-1/source/worddocuments/_24.htm	Swadesh1964		This is the famous [100 item Swadesh list](:ref:Swadesh-1955-100) in the first Russian translation. This list contains the 92 most stable items from the [master list of 215 items](:ref:Swadesh-1960-215) plus eight additional items which are only mentioned in the text and are indicated here by starring the English concept label.	78-87	Swadesh-100-Russian
+Swadesh-1960-215	Swadesh, Morris	1964		215	basic	Russian,English	Global	http://www.classes.ru/grammar/148.new-in-linguistics-1/source/worddocuments/_24.htm	Swadesh1964		This is the first Russian translation of the master list of [Swadesh's paper from 1955](:bib:Swadesh1955), which turned out to be very influential in Soviet linguistics and has retained influential up to today. Note that while the translation of the concept labels is literal and also provides the original ones which Swadesh provided in his c[master list of 215 concepts](:ref:Swadesh-1955-215), the Russian translation makes certain distinctions which cannot be determined from the original text. In one case, in the case of “fat”, this distinction is essentially wrong, as Swadesh did explicitly annotate this concept as a body part or substance. The concepts whose English labels are preceded by an asterisk are those 92 concepts which were later included in the famous [100-item list](:ref:Swadesh-1955-100).	78-87
```

Comment:
Shouldn't the changed year carry over to the YEAR column as well?

Reply:
You are right, thanks!